src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/metrics/GatkReadFilter.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.hellbender.tools.spark.pipelines.metrics;
+
+import org.apache.spark.api.java.function.Function;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+class GatkReadFilter {
```

Comment:
I'd prefer this placed in the ReadFilterLibrary class. You can name it something like 'MetricsFilter' or something.